docs/get-started/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Get Started
+lang: en-US
+sidebar: false
+layout: home
+hideHero: true
+---
+
+<div class="getstarted">
+  <div class="center">
+    <h1> Build on Ethereum, the Open Development Platform.
```

Comment:
I'm very unsure about this title, but not 100% sure what to replace it with.

"The decentralized development platform"
"Build on Ethereum"
"Learn how to build on Ethereum"
"Try Building on Ethereum"

?

Reply:
- agree "open development platform" is too vague
- I think it makes more sense to orient this text around just trying to get the user to try out the studio, rather than explain what Ethereum is

How about just:

"Get started building on Ethereum"

"Learn the basics of development on Ethereum with Ethereum Studio, a web based IDE where you can code, test, and deploy basic applications"

Reply:
Perfect, it was almost too obvious…